CalculatorWithTests/Calculator.cs

```diff
@@ -1,8 +1,38 @@
-﻿
+﻿using System;
 namespace CalculatorWithTests
 {
-    class Calculator
+    public class Calculator
     {
-        
+        public int AddTwoNumbers(int num1, int num2)
+        {
+            return num1 + num2;
+        }
+
+        public double SubtractTwoNumbers(double minuend, double subtrahend)
+        {
+            return minuend - subtrahend;
+        }
+
+        public long MultiplyTwoNumbers(long factor1, long factor2)
+        {
+            return factor1 * factor2;
+        }
+
+        public decimal DivideTwoNumbers(decimal denominator, decimal numerator)
+        {
+            return denominator / numerator;
```

Comment:
These terms are flipped. The numerator is on top, and the denominator is on the bottom: `return numerator / denominator;`